app/controllers/application_controller.rb

```diff
@@ -32,7 +32,7 @@ def squash_multiparameter_datetime_attributes(params, attribute_names)
   end
 
   def notify_bad_request(exception)
-    if %w[integration staging].include?(ENV["SENTRY_CURRENT_ENV"]) && exception.message =~ /team-only API key/
+    if %w[integration staging].include?(ENV["GOVUK_ENVIRONMENT_NAME"]) && exception.message =~ /team-only API key/
```

Comment:
Ideally we'd control this via log level rather than having the app change its behaviour based on what environment it's being told it's in, but 🤷

Reply:
A less yak-shavy compromise might be to add a `LOG_SENSITIVE_INFO` flag that defaults off and is enabled in non-prod, but that would still mean faffing about with govuk-puppet so 🤷

Reply:
Goes without saying that this isn't your fault - the problem was basically already there so it's not really getting any worse. I'm just fretting about the precedent that it sets. It's fine though 😅

Reply:
I'm honestly ok with this as is. Just don't want people thinking it's a good idea to add more logic like this 😅